core/config/locales/en.yml

```diff
@@ -69,6 +69,7 @@ en:
     invalid_volume_destination: Invalid volume specification '%{spec}' destination can't be '/'
     required_start_commands: "When 'test' is a list the first item must be one of: '%{available_commands}'"
     volumes_should_be_array: Volumes '%{volumes}' should be an arra
+    invalid_healthcheck_option: "'test' or 'disable' option must be present in healthcheck" 
```

Comment:
Maybe `required_healthcheck_options`
And let set options as string, like: `"One of these options is required: %{required_options}"